pseudocode.md

```diff
@@ -1,13 +1,66 @@
 # Pseudocode
 
-#### Add the steps you need to take to implement the requirements for Scrabble Challenge
+Add the steps you need to take to implement the requirements for Scrabble Challenge
 
-```
-1.
+## 1. Making function `findWordScore`
```

Comment:
It's really cool that you outlined this, I flat out didn't do that but it's probably really good to think about it before starting

Reply:
Thanks! Code writing is a lot easier if you don't have to worry about what the code is meant to be doing. I'd highly recommend trying it!